Dockerfile

```diff
@@ -18,6 +18,9 @@ ARG AIRFLOW_DEPS=""
 ARG PYTHON_DEPS=""
 ENV AIRFLOW_HOME=${AIRFLOW_USER_HOME}
 
+# Papermill
+ARG PAPERMILL_PACKAGE=""
```

Comment:
default this to "papermill"

Reply:
sounds good

Reply:
@devstein just to check - i did this correctly? im not a moron am i?